CHANGELOG.rst

```diff
@@ -15,6 +15,10 @@ Unreleased
 ~~~~~~~~~~
 * Add RST validator
```

Comment:
NIT: could you add this bullet point to the release

Reply:
Whoops, missed that in my rebase. Thank you.